group_vars/whichiso/main.yml

```diff
@@ -29,6 +29,8 @@ project_db_host: '{{postgres_host}}'
 install_ruby_from_source: true
 ruby_version_override: "ruby-3.2.4"
 passenger_ruby: "/usr/local/bin/ruby"
+passenger_extra_http_config:
```

Comment:
Discussed in Ansible open hours: We believe this line may not be needed; this should be tested when @Beck-Davis gets back to looking at this PR.